src/components/Contentful/StoryCard.vue

```diff
@@ -32,6 +67,9 @@ export default {
 		theme() {
 			return this.content?.theme ?? '';
 		},
+		themeClass() {
+			return `story-card__${this.content?.theme}`;
```

Comment:
Temporary theme class created to handle new card colors. TODO review and implement theming with @ryan-ludwig